arboretum/kubernetes/README.md

```diff
@@ -16,7 +16,80 @@ to include the fetchers and checks from this library in your downstream project.
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster Resource
+
+* Class: [ClusterResourceFetcher][fetch-cluster-resource]
+* Purpose: Write the resources of **stand-alone** Kubernetes clusters to the
+evidence locker.  **NOTE:** Do not use this fetcher for managed clusters.
+Instead use the [IBM Cloud cluster list fetcher][ibm-cloud-cluster-list-fetcher].
+* Behavior: Retrieve stand-alone Kubernetes cluster resource data for the provided
+list of clusters.  TTL is set to 1 day.
+* Configuration elements:
+  * `org.kubernetes.cluster_resources.clusters`
+    * Required
+    * List of dictionaries:
+      * `label`: An arbitrary label to specify token in credentials file
+      * `server` A URL as a string to a Kubernetes stand-alone cluster
+  * `org.kubernetes.cluster_resources.types`
+    * Optional
+    * List of resource types as strings
+      * NOTE: For core group API resources, the resource name must be in
+      _plural form_.  See example below.
+      * NOTE: For other "custom" API resources, the resource name must be in the
+      following format: `"APIGROUP/VERSION/NAME"`.  See example below.
+    * Use if looking to override the default of `["nodes", "pods", "configmaps"]`.
+    Otherwise do not include.
+* Expected configuration:
+
+  ```json
+  {
+    "org": {
+      "kubernetes": {
+        "cluster_resources": {
+          "clusters": [
+            {
+              "label": "devcluster1",
+              "server": "https://myserver1:30000"
+            }
+          ],
+          "types": ["nodes", "pods", "apigroup.example.com/v1/mycustom"]
+        }
+      }
+    }
+  }
+   ```
+
+* Required credentials:
+  * A token of a kubernetes service account is required. Following example
+    shows how to create a token bound with a service account.
+
+    ```sh
+    $ kubectl create serviceaccount auditree_account # this creates both serviceaccount and its secret
+    $ kubectl get serviceaccount auditree_account -ojsonpath='{.secrets[0].name}' | xargs kubectl get secret -ojsonpath='{.data.token}'
+    ZXlKaG... # use this string as a token
+    ```
+
+    In addition, appropriate `get` verbs should be permitted to the service
+    account. For example, when the fetcher is configured to fetch `pod`
+    resources from a cluster, permit `get` verb for `pod` resources of all
+    namespaces in the cluster to the service account by creating
+    `clusterrole` and `clusterrolerolebinding`.  For details, see the
+    [Kubernetes RBAC documentation](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+    * `XXX_token`: token string for a cluster named `XXX`. For example, a token
+      with key `devcluster1_token` is used for the cluster `devcluster1`.
```

Comment:
This is still unclear.  I _think_ you mean that the cluster's "label" as it is set in the configuration is what "XXX" correlates to.  Suggest changing this to:

```suggestion
    * `cluster_label_token`: a token string for the cluster label value from your configuration. Using the expected 
    configuration from above as an example, a token named `devcluster1_token` is an expected credential.
```